
From: Yoichi Yuasa <yuasa@hh.iij4u.or.jp>

This patch had added missing definition and had fixed typo for VRC4173.

Cc: Ralf Baechle <ralf@linux-mips.org> 
Signed-off-by: Yoichi Yuasa <yuasa@hh.iij4u.or.jp>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/mips/vr41xx/common/vrc4173.c |    5 ++++-
 25-akpm/include/linux/pci_ids.h           |    1 +
 2 files changed, 5 insertions(+), 1 deletion(-)

diff -puN arch/mips/vr41xx/common/vrc4173.c~mips-added-missing-definition-and-fixed-typo arch/mips/vr41xx/common/vrc4173.c
--- 25/arch/mips/vr41xx/common/vrc4173.c~mips-added-missing-definition-and-fixed-typo	2004-10-03 16:44:04.612508960 -0700
+++ 25-akpm/arch/mips/vr41xx/common/vrc4173.c	2004-10-03 16:44:04.617508200 -0700
@@ -56,6 +56,9 @@ MODULE_LICENSE("GPL");
 
 #define VRC4173_SYSINT1REG	0x060
 #define VRC4173_MSYSINT1REG	0x06c
+#define VRC4173_MPIUINTREG	0x06e
+#define VRC4173_MAIUINTREG	0x070
+#define VRC4173_MKIUINTREG	0x072
 
 #define VRC4173_SELECTREG	0x09e
  #define SEL3			0x0008
@@ -329,7 +332,7 @@ void vrc4173_enable_piuint(uint16_t mask
 	spin_unlock_irqrestore(&desc->lock, flags);
 }
 
-EXPORT_SYMBOL(vrc4173_eanble_piuint);
+EXPORT_SYMBOL(vrc4173_enable_piuint);
 
 void vrc4173_disable_piuint(uint16_t mask)
 {
diff -puN include/linux/pci_ids.h~mips-added-missing-definition-and-fixed-typo include/linux/pci_ids.h
--- 25/include/linux/pci_ids.h~mips-added-missing-definition-and-fixed-typo	2004-10-03 16:44:04.614508656 -0700
+++ 25-akpm/include/linux/pci_ids.h	2004-10-03 16:44:04.619507896 -0700
@@ -569,6 +569,7 @@
 #define PCI_DEVICE_ID_NEC_PCX2		0x0046 /* PowerVR */
 #define PCI_DEVICE_ID_NEC_NILE4		0x005a
 #define PCI_DEVICE_ID_NEC_VRC5476       0x009b
+#define PCI_DEVICE_ID_NEC_VRC4173	0x00a5
 #define PCI_DEVICE_ID_NEC_VRC5477_AC97  0x00a6
 #define PCI_DEVICE_ID_NEC_PC9821CS01    0x800c /* PC-9821-CS01 */
 #define PCI_DEVICE_ID_NEC_PC9821NRB06   0x800d /* PC-9821NR-B06 */
_
